examples/host_commands.h

```diff
@@ -271,4 +273,16 @@ struct ec_channel_uart_config_set_req {
   struct ec_channel_uart_config config;
 } __attribute__((packed, aligned(4)));
 
+#define EC_CMD_CONSOLE_REQUEST 0x0097
+#define EC_CMD_CONSOLE_READ 0x0098
+
+enum ec_console_read_subcmd {
+	CONSOLE_READ_NEXT = 0,
+	CONSOLE_READ_RECENT
```

Comment:
nit: `CONSOLE_READ_RECENT = 1,`, as this is a protocol and the actual enum values must be stable.

(Yes, this is also missing in the header you consulted to add here, which we should also fix.)

Reply:
Done.
It seems this file already contains contents prefixed with `EC_` that are copied-and-pasted from the header file I consulted to (ec_header.h), so I assumed "copying what is needed to this file" is the usual practice.